eclipse-collections/src/main/java/org/eclipse/collections/impl/multimap/bag/SynchronizedPutHashBagMultimap.java

```diff
@@ -175,4 +175,10 @@ public void putOccurrences(K key, V value, int occurrences)
             }
         }
     }
+
+    @Override
+    public MutableBagMultimap<K, V> asSynchronized()
+    {
+        throw new UnsupportedOperationException("Cannot call asSynchronized() on SynchronizedPutMultimap");
```

Comment:
To keep the exception consistent it should be:
`throw new UnsupportedOperationException("Cannot call asSynchronized() on " + this.getClass().getSimpleName())`